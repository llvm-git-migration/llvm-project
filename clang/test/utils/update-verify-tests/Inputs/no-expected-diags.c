// RUN: %clang_cc1 -verify %s
// RUN: diff %s %s.expected
// expected-no-diagnostics
void foo() {
    a = 2;
}
