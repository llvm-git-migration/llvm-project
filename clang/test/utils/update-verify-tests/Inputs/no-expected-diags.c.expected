// RUN: %clang_cc1 -verify %s
// RUN: diff %s %s.expected
void foo() {
    // expected-error@+1{{use of undeclared identifier 'a'}}
    a = 2;
}
